app/templates/app/src/main/java/screen/main/_MainActivity.java

```diff
@@ -1,408 +0,0 @@
-        }
-
-        actionBar.setDisplayHomeAsUpEnabled(config.isHomeAsUpEnabled());
-    }
-
-    @Override
-    public Toolbar getToolbar() {
-        return toolbar;
-    }
-
-    /**
-     * Dev tools and the play store (and others?) launch with a different intent, and so
-     * lead to a redundant instance of this activity being spawned. <a
-     * href="http://stackoverflow.com/questions/17702202/find-out-whether-the-current-activity-will-be-task-root-eventually-after-pendin"
-     * >Details</a>.
-     */
-    private boolean isWrongInstance() {
-        if (!isTaskRoot()) {
-            Intent intent = getIntent();
-            boolean isMainAction = intent.getAction() != null && intent.getAction().equals(ACTION_MAIN);
-            return intent.hasCategory(CATEGORY_LAUNCHER) && isMainAction;
-        }
-        return false;
-    }
-
-    /**
-     * Configure the app activity bar
-     */
-    private void configureToolbar() {
-        setSupportActionBar(toolbar);
-
-        menuItemSelectionHandlers = map(
-                android.R.id.home, new UpSelectionHandler(),
-                R.id.action_environment, new SwitchEnvironmentSelectionHandler()
-        );
-
-        toolbarOwner = new ToolbarOwner();
-        toolbarOwner.takeView(this);
-    }
-
-    /**
-     * Inflate the view container layout and inject our view components
-     */
-    private void inflateViewContainerLayout() {
-        setContentView(R.layout.activity_main);
-        ButterKnife.bind(this);
-    }
-
-    /**
-     * Initalise the root activity Mortar scope
-     */
-    protected void initActivityScope(Bundle savedInstanceState) {
-        mortarScope = MortarScope.findChild(getApplicationContext(), getClass().getName());
-
-        if (mortarScope == null) {
-            component = DaggerActivityComponent.builder()
-                    .applicationComponent(DaggerService.<ApplicationComponent>getDaggerComponent(getApplicationContext()))
-                    .activityModule(new ActivityModule(this))
-                    .build();
-
-            mortarScope = MortarScope.buildChild(getApplicationContext())
-                    .withService(BundleServiceRunner.SERVICE_NAME, new BundleServiceRunner())
-                    .withService(DaggerService.SERVICE_NAME, component)
-                    .build(getClass().getName());
-        }
-    }
-
-
-    private int getToolbarHeight() {
-        TypedArray styledAttributes = getTheme().obtainStyledAttributes(new int[]{android.R.attr.actionBarSize});
-        int height = (int) styledAttributes.getDimension(0, 0);
-        styledAttributes.recycle();
-        return height;
-    }
-
-    private class UpSelectionHandler implements MenuItemSelectionHandler {
-        @Override
-        public boolean execute() {
-            return containerAsHandlesBack.onBackPressed();
-        }
-    }
-
-    private class SwitchEnvironmentSelectionHandler implements MenuItemSelectionHandler {
```

Comment:
How about we enable Dev Options (where we could switch environment) somewhere else other than the toolbar? Maybe we can even trigger Dev Options through a shake?